### DISCUSSION TITLE
Modo Release don't get the data

### DISCUSSION THREAD

**Discussion by PabloRuiz-TT** [Azure/azure-mobile-apps#919] [Q&A]
Hello, I have an application which I declare my tables with the IOfflineTable<> interface. In debug mode it works without problems, it brings me the data from the api but when I implement my solution in release mode, I don't get the data. Any idea ?

**Comment by adrianhall** [comment]
Short version - nope, no idea.

However, you have not provided any details of the application, how you are calling the push/pull, what the differences are between debug and release, etc.  With so little information, it's impossible to suggest a way forward.
